doc/releases/octopus.rst

```diff
@@ -1,3 +1,72 @@
+v15.2.10 Octopus
+================
+
+This is the 10th backport release in the Octopus series. We recommend all
+users update to this release.
+
+Notable Changes
+---------------
+
+* The containers include an updated tcmalloc that avoids crashes seen on 15.2.9.  See `issue#49618 <https://tracker.ceph.com/issues/49618>`_ for details.
+
+* When upgrading from a previous cephadm release, please redeploy iSCSI, NFS and the monitoring stack. Please run the following to upgrade the rest of the services.::
```

Comment:
@sebastian-philipp @liewegas would be helpful to mention why a user should do this

Reply:
I'm just waiting on this.  15.2.10 is built, signed, and ready to push.

Reply:
The cephadm systemd unit changed to ``type=forking``, which requires the ``/var/lib/ceph/$fsid/$name/unit.run`` files to be regenerated with a ``-d`` flag.  Otherwise, a ``systemctl start`` will hang and time out.

Reply:
When upgrading from a previous cephadm release, ``systemctl`` may hang when trying to start or restart the monitoring containers.  (This is cause by a change in the systemd unit to use ``type=forking``.)  After the upgrade, please run::

   ceph orch redeploy prometheus
   ...